k8s/README.md

```diff
@@ -0,0 +1,51 @@
+# Nanokube
+
+Nanokube is Nanotube on Kubernetes.
+
+Nanotube can be run on kubernetes as a daemonset; each of the pods of nanokube daemonset accept and receive the traffic
```

Comment:
Minor note: Please consider merging lines in a single paragraph in the md files.